File: src/OnPeople/BackEnd/src/OnPeople.Domain/Models/Contas/Conta.cs
using OnPeople.Domain.Models.Empresas;

namespace OnPeople.Domain.Models.Contas
{
    public class Conta
    {
        public int Id { get; set; }
        public string NomeCompleto { get; set; }
        public string Visao { get; set; }
        public string Foto { get; set; }
        public DateTime DataCadastro { get; set; }
        public DateTime? DataEncerramento { get; set; }
        public Boolean Ativa { get; set; }
        IEnumerable<ContaFuncao> ContasFuncoes {get; set;}
        IEnumerable<Empresa> Empresas {get; set;}
    }
}

File: src/OnPeople/FrontEnd/OnPeople-App/src/app/models/contas/Conta.ts
import { Empresa } from "../empresas/Empresa";
import { ContaFuncao } from "./ContaFuncao";


export interface Conta {
  id: number;
  nomeCompleto: string;
  visao: string;
  foto: string;
  dataCadastro: Date;
  dataEncerramento?: Date;
  ativa: Boolean;
  contasFuncoes: ContaFuncao;
  empresas: Empresa;
}
